src/Features/LanguageServer/Protocol/Handler/Diagnostics/AbstractPullDiagnosticHandler.cs

```diff
@@ -320,27 +325,28 @@ private void HandleRemovedDocuments(RequestContext context, ImmutableArray<Previ
             }
         }
 
-        private ImmutableArray<LSP.Diagnostic> ConvertDiagnostic(IDiagnosticSource diagnosticSource, DiagnosticData diagnosticData, ClientCapabilities capabilities)
+        private ImmutableArray<LSP.Diagnostic> ConvertDiagnostic(
+            IDiagnosticSource diagnosticSource,
+            DiagnosticData diagnosticData,
+            SourceText? sourceText,
+            ClientCapabilities capabilities)
         {
+            if (diagnosticData.IsSuppressed)
+                return ImmutableArray<LSP.Diagnostic>.Empty;
```

Comment:
this check was missing in LSP, but is included in our normal tagger.  included it to keep things consistent.